next-app/src/routes/projects/[project_code]/activities/+server.ts

```diff
@@ -1,20 +1,22 @@
 import { json } from '@sveltejs/kit'
 import { sf } from '$lib/server/sf'
+import type { Rpc } from '$lib/server/types'
+import type { RequestHandler } from './$types'
 
-export async function GET({ params: { project_code }, request: { headers } }) {
-	const cookie = headers.get('cookie')
+export const GET = (async ({ params: { project_code }, request: { headers } }) => {
```

Comment:
same here as on password server, we can use RequestEvent instead and preserve this as a function.